gdcdictionary/schemas/treatment.yaml

```diff
@@ -130,8 +199,14 @@ properties:
     $ref:
       - "_terms.yaml#/route_of_administration/common"
     enum:
+      - Intrahepatic
+      - Intramuscular
+      - Intraperitoneal
       - Intrathecal
+      - Intratumor
```

Comment:
@tlicht3 , should this be "Intratumoral"?  Google seems to think so, but Google is often wrong.

Reply:
The ticket DICT-174 indicates 'Intratumoral'.

Reply:
fixed in ticket and here.